README.md

```diff
@@ -1,67 +1,66 @@
-﻿# Project Reunion
+﻿# Project Reunion - Calling all Windows desktop apps!

```

Comment:
should this say "Windows desktop apps" or just "Windows apps"?

Reply:
Currently we are focusing on and prioritizing Windows desktop apps.

Reply:
agreed, but you do mention UWP apps further below. like jonwis said, since this is just a point-in-time thing, it seems like this could cause confusion